docs/CONFIGURATION-AS-CODE.md

```diff
@@ -8,30 +8,32 @@
 
 [Definition](https://github.com/jenkinsci/ec2-fleet-plugin/blob/master/src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java#L156-L179)
 
-| Property        | Type           | Required  | Description |
-|-------------------|---|---|---|
-| name              |string|yes|ec2-fleet|
-| awsCredentialsId  |string|no, default ```null```|[Leave blank to use AWS EC2 instance role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html)|
-| computerConnector |object|yes|for example ```sshConnector```|
-| region            |string|yes|```us-east-2```, full [list](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html)|
-| fleet   |string|yes|my-fleet|
-| endpoint|string|no|Set only if you need to use custome endpoint ```http://a.com```|
-| fsRoot  |string|no|my-root|
-| privateIpUsed|boolean|no, default ```false```|connect to EC2 instance by private id instead of public|
-| alwaysReconnect|boolean|no, default ```false```|if ```true``` always reconnect to offline nodes after instance reboot or connection loss|
-| labelString |string|yes||
-| idleMinutes |int|no, default ```0```||
-| minSize     |int|no, default ```0```||
-| maxSize     |int|no, default ```0```||
-| numExecutors|int|no, default ```1```||
-| addNodeOnlyIfRunning|boolean|no, default ```false```||
-| restrictUsage|boolean|no, default ```false```|if ```true``` fleet nodes will executed only jobs with same label|
-| scaleExecutorsByWeight|boolean|no, default ```false```||
-| initOnlineTimeoutSec|int|no, default ```180```||
-| initOnlineCheckIntervalSec|int|no, default ```15```||
-| cloudStatusIntervalSec|int|no, default ```10```||
-| disableTaskResubmit|boolean|no, default ```false```||
-| noDelayProvision|boolean|no, default ```false```||
+| Property                   | Type    | Required                          | Description |
+|----------------------------|---------|-----------------------------------|---|
+| name                       | string  | yes                               |ec2-fleet|
```

Comment:
Is this description of name meaningful? I think we should explain a little bit here since we're making changes to make names unique. Maybe something like "Name used to track Clouds. Empty string provides a default name."